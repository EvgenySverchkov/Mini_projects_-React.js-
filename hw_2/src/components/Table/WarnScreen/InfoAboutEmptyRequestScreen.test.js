import React from 'react';
import { shallow, render } from 'enzyme';

import ErrMessage from './InfoAboutEmptyRequestScreen';

const wrapper = render(<ErrMessage />);
it('Snapshot tets for <InfoAboutEmptyRequestScreen />', ()=>{
	expect(wrapper).toMatchSnapshot();
});