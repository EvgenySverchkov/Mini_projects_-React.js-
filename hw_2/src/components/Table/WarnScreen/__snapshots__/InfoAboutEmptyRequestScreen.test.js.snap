// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tets for <InfoAboutEmptyRequestScreen /> 1`] = `
<div
  className="jumbotron jumbotron-fluid"
>
  <div
    className="container"
  >
    <h1
      className="display-4"
    >
      Sorry...
    </h1>
    <p
      className="lead"
    >
      User not found
    </p>
  </div>
</div>
`;
